Extensions/CodeGeneration/CodeGenerationDisplayDriver.cs

```diff
@@ -133,16 +133,18 @@ private string ConvertJToken(JToken jToken)
                     };
 
                 case JArray jArray:
-                    var token = string.Join(", ", jArray.Select(ConvertJToken));
-                    var format = token.Contains("ListValueOption") ? "\n    " : string.Empty;
-
+                    var token = string.Join(", ", jArray.Select(a => ConvertJToken(a, indentationDepth * 2)));
+                    var format = token.Contains("ListValueOption", StringComparison.InvariantCulture)
+                        ? $"\n{string.Join(string.Empty, Enumerable.Repeat(" ", (int)(indentationDepth * 1.5)))}"

```

Comment:
Why `1.5`?

Reply:
Without it, the array initialization braces were misaligned. The code that calls this sends 8 as the `indentationDepth`. `1.5` gives us 12 spaces.

![image](https://user-images.githubusercontent.com/237369/157139503-b9339b24-752c-4d67-81b9-e1713920517d.png)

Reply:
Why not simply add 4? Note that what's provided in `indentationDepth` might not be 8, and this can be deeper.

Reply:
I converted it to add 4 last night.